content/en/docs/pipelines/installation/overview.md

```diff
@@ -4,41 +4,37 @@ description = "Overview of the ways to deploy Kubeflow Pipelines"
 weight = 10
                     
 +++
-{{% alert title="Out of date" color="warning" %}}
-This guide contains outdated information pertaining to Kubeflow 1.0. This guide
-needs to be updated for Kubeflow 1.1.
-{{% /alert %}}
 
 Kubeflow Pipelines offers a few installation options.
 This page describes the options and the features available
 with each option:
 
-* A [standalone](#standalone) Kubeflow Pipelines deployment is the minimal
+* [Kubeflow Pipelines Standalone](#kubeflow-pipelines-standalone) is the minimal
 portable installation that only includes Kubeflow Pipelines.
-* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow) provides
+* Kubeflow Pipelines as [part of a full Kubeflow deployment](#full-kubeflow-deployment) provides
 all Kubeflow components and more integration with each platform.
-* **Beta**: [Google Cloud AI Platform Pipelines](#marketplace) provides easy-to-use management
-UI for installing and using Kubeflow Pipelines on Google Cloud.
+* **Beta**: [Google Cloud AI Platform Pipelines](#google-cloud-ai-platform-pipelines) provides easy-to-use management
+UI for installing and using Kubeflow Pipelines Standalone on Google Cloud.
 
 ## Choosing an installation option
 
 1. Do you want to use other Kubeflow components in addition to Pipelines?
 
-    If yes, choose the [full Kubeflow](#full-kubeflow).
+    If yes, choose the [full Kubeflow](#full-kubeflow-deployment).
 1. Do you want to use Kubeflow Pipelines with [multi-user support](https://github.com/kubeflow/pipelines/issues/1223)?
 
-    If yes, wait for the feature to be released in [Kubeflow](#full-kubeflow) 1.1.
+    If yes, choose the [full Kubeflow](#full-kubeflow-deployment) with version >= v1.1.
 1. Do you deploy on Google Cloud?
 
-    If yes, deploy [Kubeflow Pipelines Standalone](#standalone). You can also
-    use [AI Platform Pipelines](#marketplace) to deploy Kubeflow Pipelines
+    If yes, deploy [Kubeflow Pipelines Standalone](#kubeflow-pipelines-standalone). You can also
+    use [Google Cloud AI Platform Pipelines](#google-cloud-ai-platform-pipelines) to deploy Kubeflow Pipelines
     Standalone using a user interface, but there are limitations in
```

Comment:
Same as above, we should not refer to AI Platform Pipelines installing KFP Standalone. Since users need to understand that KFP Standalone is a separate thing.

Reply:
Good point! Thanks